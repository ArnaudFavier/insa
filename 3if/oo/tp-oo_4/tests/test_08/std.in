S segment -3 1 5 1
R rectangle -8 4 4 -4
PC poly 1 1 -1 2 -3 0 -1 -3 1 0
OI inter1 segment rectangle poly
LIST
MOVE segment 10 10
OI inter2 inter1 segment rectangle poly
LIST
HIT inter1 -1 1
HIT inter1 1 1
HIT inter1 -2 0
HIT inter1 -8 4
EXIT
