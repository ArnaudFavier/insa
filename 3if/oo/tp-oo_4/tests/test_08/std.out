OK
OK
OK
OK
inter1 I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
OK
OK
inter1 I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
inter2 I [I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]] S [(7;11) (15;11)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(7;11) (15;11)]
YES
YES
NO
NO
